### PR TITLE
[TACHYON-1577] Clean up HeartbeatExecutor threadpools

### PR DESCRIPTION
**Pull request by aaudiber**
https://tachyon.atlassian.net/browse/TACHYON-1577

### CODE REVIEW

File: common/src/main/java/tachyon/heartbeat/HeartbeatExecutor.java [hunk 1]
```diff
@@ -15,13 +15,20 @@
...
+public interface HeartbeatExecutor extends Closeable {
   /**
    * Implements the heartbeat logic.
    */
   void heartbeat();
+
+  /**
+   * Cleans up any resources used by the heartbeat executor.
+   */
+  void close();
```
Comment (apc999) [inline common/src/main/java/tachyon/heartbeat/HeartbeatExecutor.java]: `@Override` ?

Comment (aaudiber) [inline common/src/main/java/tachyon/heartbeat/HeartbeatExecutor.java]: Updated, thanks

**Comment by yupeng9** [comment]
Jenkins, test this please

**Comment by yupeng9** [comment]
LGTM

**Comment by apc999** [comment]
LGTM

**Comment by aaudiber** [comment]
Jenkins, test this please.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**